docker-compose.yml

```diff
@@ -1,12 +1,18 @@
-version: "2.2"
-
+version: "3"
+name: "dj"
```

Comment:
Probably don't need to quote the `dj` value here.

Reply:
Fixed 👍